administration/src/cards/CardBlueprint.ts

```diff
@@ -51,16 +52,17 @@ export class CardBlueprint {
   }
 
   hasValidSize(): boolean {
-    // every four characters in a base64 encoded string correspond to three bytes
-    // the qr code can hold 224 characters
-    // therefore 224 / 4 * 3 = 168
     // See https://github.com/digitalfabrik/entitlementcard/issues/690  for more context.
-    return this.sizeOfProtobuf() <= 168
-  }
 
-  sizeOfProtobuf(): number {
-    return new QrCode({ qrCode: { value: this.generateActivationCode(), case: 'dynamicActivationCode' } }).toBinary()
-      .length
+    const dynamicCode = new QrCode({
+      qrCode: { value: this.generateActivationCode(), case: 'dynamicActivationCode' },
+    }).toBinary()
+
+    const staticCode = new QrCode({
+      qrCode: { value: this.generateStaticVerificationCode(), case: 'dynamicActivationCode' },
```

Comment:
```suggestion
      qrCode: { value: this.generateStaticVerificationCode(), case: 'staticVerificationCode' },
```
Wild, that this does not produce a typescript error. Can you check why?